internal/exec/stages/disks/partitions.go

```diff
@@ -320,9 +320,12 @@ func (p PartitionList) Swap(i, j int) {
 func (s stage) partitionDisk(dev types.Disk, devAlias string) error {
 	if dev.WipeTable != nil && *dev.WipeTable {
 		op := sgdisk.Begin(s.Logger, devAlias)
+		//nolint:errcheck
 		s.Logger.Info("wiping partition table requested on %q", devAlias)
 		op.WipeTable(true)
-		op.Commit()
+		if err := op.Commit(); err != nil {
```

Comment:
This is the kind of fix I'd split out separately, missing an error check on a `Commit()` call seems like an important fix.